SPEC.md

```diff
@@ -567,6 +567,7 @@ Plugins must output a JSON object with the following keys upon a successful `ADD
 - `interfaces`: An array of all interfaces created by the attachment, including any host-level interfaces:
     - `name` (string): The name of the interface.
     - `mac` (string): The hardware address of the interface (if applicable).
+    - `mtu`: The MTU of the interface (if applicable).
```

Comment:
Could you add type in the fields? i.e. `(uint)`

Reply:
nice catch thanks!